api/commands.py

```diff
@@ -503,12 +501,12 @@ def add_qdrant_doc_id_index(field: str):
         from core.rag.datasource.vdb.qdrant.qdrant_vector import QdrantConfig
         for binding in bindings:
             qdrant_config = QdrantConfig(
-                endpoint=config.get('QDRANT_URL'),
-                api_key=config.get('QDRANT_API_KEY'),
+                endpoint=dify_config.QDRANT_URL or '',
```

Comment:
We'd better not to have this patch for default value, not elegant as expected. Use proper default values like Optional or empty string in config entry definition to handle similar situations.

Reply:
I fixed it, thank you.

Reply:
Thansk. LGTM.